tests/test_vlan.py

```diff
@@ -447,13 +447,10 @@ def test_AddPortChannelToVlan(self, dvs, testlog):
         assert len(lag_member_entries) == 1
 
         (status, fvs) = tbl.get(lag_member_entries[0])
-        for fv in fvs:
-            if fv[0] == "SAI_LAG_MEMBER_ATTR_LAG_ID":
-                assert fv[1] == lag_entries[0]
-            elif fv[0] == "SAI_LAG_MEMBER_ATTR_PORT_ID":
-                assert dvs.asicdb.portoidmap[fv[1]] == "Ethernet0"
-            else:
-                assert False
+        fvs = dict(fvs)
+        assert len(fvs) == 4
```

Comment:
IMO, I dont think you should check for `len(fvs) == 4(hardcoded)` but check for individual attributes in the `if` case. @daall , any suggestions?

Reply:
It's currently hard-coded in the master as well, but I do agree that it is better to check for individual attributes. I think that can be taken up as a separate PR though since this issue also exists in master.